app/views/full-page-examples/passport-details/index.js

```diff
@@ -8,7 +8,7 @@ module.exports = (app) => {
       body('passport-number')
         .exists()
         .not().isEmpty().withMessage('Enter your passport number'),
-      body('expiry-day')
+      body('expiry')
```

Comment:
I'm not convinced making these changes on the date input component actually makes it any easier.

You can see that I was not able to simplify the logic in the validation section: https://github.com/alphagov/govuk-frontend/blob/d41467da8dafd2b259cd06daa66bf4a7266a9d5d/app/views/full-page-examples/passport-details/index.js#L28-L52

Is it worth changing the date input considering this?

Reply:
Noticed that the full page example that features the date input doesnt work anymore, so maybe I will have to change this logic, will see if it makes it simpler or not, I suspect it'll be more complex than it already is...